addons/l10n_ch/models/res_bank.py

```diff
@@ -272,7 +272,8 @@ def _is_qr_iban(self):
         QR-codes. They are formed like regular IBANs, but are actually something
         different.
         """
-        self.ensure_one()
+        if not self:
```

Comment:
```suggestion
        if len(self) == 1:
```
self.ensure_one() does more than checking if we have a record, it also makes sure we have only one.

Reply:
And actually, I re-read myself here, you are going to raise the error if we have one bank account which is the expected use case. You will want to raise it if len(self) does not contain one and only one record.